run_controller_init_setup/run_controller_init_setup.py

```diff
@@ -1464,15 +1481,16 @@ def run_initial_setup(
         print(indent + keyword_for_log + "Request Method Type : " + str(request_method))
         print(indent + keyword_for_log + "Request payload     : \n" + str(json.dumps(obj=data, indent=4)))
 
-        response = _send_aviatrix_api(
-            api_endpoint_url=api_endpoint_url,
-            request_method=request_method,
-            payload=data,
-            keyword_for_log=keyword_for_log,
-            indent=indent + "    "
-        )
-        return response
-    except requests.exceptions.ConnectionError as err:
+        with timeout(limit=600):

```

Comment:
You could just use the timeout argument in requests.post

Reply:
The problem is that the `request.post` is in the function `_send_aviatrix_api`, and it will retry 5 times, then raise an `AviatrixException` finally no matter what exception it is. I think it's not very specific to this issue. What do you think?

Reply:
That does not seem to be right, We should not retry intial setup run API. This API should only be called once